src/govuk/components/character-count/character-count.mjs

```diff
@@ -91,19 +107,28 @@ CharacterCount.prototype.getDataset = function (element) {
   return dataset
 }
 
-// Counts characters or words in text
+/**
+ * Count the number of characters (or words, if `options.maxwords` is set)
+ * in the given text
+ *
+ * @param {String} text - The text to count the characters of
+ * @returns {Integer} the number of characters (or words) in the text
+ */
 CharacterCount.prototype.count = function (text) {
-  var length
   if (this.options.maxwords) {
     var tokens = text.match(/\S+/g) || [] // Matches consecutive non-whitespace chars
-    length = tokens.length
+    return tokens.length
   } else {
-    length = text.length
+    return text.length
   }
-  return length
 }
 
-// Bind input propertychange to the elements and update based on the change
+/**
+ * Bind change events
+ *
+ * Set up events on the $textarea so that the counts update when the user
```

Comment:
This makes it sound like internal counts are updating I think.

"... count messages update..."?